sensors/src/loadbalancer_sensor.py

```diff
@@ -0,0 +1,167 @@
+import subprocess
+import logging
+import requests
+from st2reactor.sensor.base import Sensor
+from st2reactor.container.sensor_wrapper import SensorService
+from amphorae import get_amphorae
+
+
+# pylint: disable=abstract-method
+class LoadbalancerSensor(Sensor):
+    def __init__(self, sensor_service: SensorService, config):
+        # pylint: disable=super-with-arguments
+        super(LoadbalancerSensor, self).__init__(
+            sensor_service=sensor_service, config=config
+        )
+        self.sensor_service: SensorService = sensor_service
+        self._logger = self.sensor_service.get_logger(name=self.__class__.__name__)
+
+    # pylint: disable=inconsistent-return-statements
+    def run(
+        self,
+        cloud_account: str = "dev-admin",
+    ):
+        """
+        Action to check loadbalancer and amphorae status
+        output suitable to be passed to create_tickets
+        """
+
+        amphorae = get_amphorae(cloud_account)
+
+        amph_json = self._check_amphora_status(amphorae)
+        # pylint: disable=line-too-long
+        output = {
+            "title": "{p[title_text]}",
+            "body": "The loadbalance ping test result was: {p[lb_status]}\nThe status of the amphora was: {p[amp_status]}\nThe amphora id is: {p[amp_id]}\nThe loadbalancer id is: {p[lb_id]}",
+            "server_list": [],
+        }
+        if amphorae.status_code != 200:
+            # Notes problem with accessing api if anything other than 403 or 200 returned
+            logging.critical("We encountered a problem accessing the API")
+            logging.critical("The status code was: %s ", str(amphorae.status_code))
+            logging.critical("The JSON response was: \n %s", str(amph_json))
+            return False
```

Comment:
`# pylint: disable=inconsistent-return-statements`
Can be removed if this line simply is
`return`
and....